partition/roles/wireguard/tasks/main.yaml

```diff
@@ -0,0 +1,99 @@
+---
+- name: check mandatory variables for this role are set
+  assert:
+    fail_msg: "not all mandatory variables given, check role documentation"
+    quiet: yes
+    that:
+      - wireguard_ip is not none
+      - wireguard_clients is not none
+      - wireguard_clients|length > 0
+
+- name: install wireguard
+  apt:
+    name: wireguard
+    update_cache: yes
+    state: latest
+  with_items:
+    - wireguard
+    - wireguard-dkms
+    - wireguard-tools
+  tags: wg-install
```

Comment:
Do we need the tags on these tasks? Somehow we do not have any tags anywhere else in the roles of this repo.